package/MDAnalysis/analysis/psa.py

```diff
@@ -575,7 +575,7 @@ def dist_mat_to_vec(N, i, j):
       int, index (of the matrix element) in the corresponding distance vector
     """
 
-    if not (isinstance(N, int) or isinstance(i, int) or isinstance(j, int)):
+    if not (isinstance(N, numbers.Integral) or isinstance(i, numbers.Integral) or isinstance(j, numbers.Integral)):
```

Comment:
@kain88-de I was thinking about how to check whether input is numpy integer or not, I think we are only checking whether the input **is not** an integer, How should I validate whether the input **is** valid integer?